.gitlab-ci-check-golang-lint.yml

```diff
@@ -24,7 +24,7 @@ test:static:
   needs: []
   except:
     - /^saas-[a-zA-Z0-9.]+$/
-  image: golangci/golangci-lint:v1.43.0
+  image: golangci/golangci-lint:v1.47.3
```

Comment:
I've been experimenting with this and the formatting will change enough to make release branches (and tags) of some repos don't pass anymore. We need an strategy on how to do this:  can we discuss it in the next QA meeting?

Reply:
Thanks. I'll hold back this PR until the release is done and we have a strategy for such changes. I will update the local CI configuration for the affected PR (mendersoftware/deployments#769).